compiler/rustc_codegen_llvm/Cargo.toml

```diff
@@ -9,6 +9,7 @@ test = false
 [dependencies]
 # tidy-alphabetical-start
 bitflags = "2.4.1"
+gimli = "0.30"
```

Comment:
mmm, three versions of gimli in the repo... should fix that...

Reply:
Yeah, it looks like we have:

- 0.31 (latest) used by run-make-support, which actually needs it
- 0.30 used by most stuff, which can probably be bumped to 0.31
- 0.28 used by an old addr2line, which is pulled in by anyhow's `backtrace` feature, which is perhaps not even needed nowadays?